src/pages/patientView/mutation/column/TumorColumnFormatter.tsx

```diff
@@ -67,4 +66,17 @@ export default class TumorColumnFormatter {
             return map;
         }, {});
     }
+
+    public static getSample(data:Array<{sampleId:string}>): string|string[] {
+        let result =[];
+        if (data) {
+            for (let datum of data) {
```

Comment:
i'm pretty sure for ... of is not supported in IE11.  there is a babel tranform for it but i don't think we have it installed and probably not worth it.  i would just use Array.forEach to be safe.  we definately already have a polyfill for that.